MAINTAINERS.md

```diff
@@ -0,0 +1,165 @@
+# Kubernetes Project Maintainership
+
+Kubernetes is a big project with many contributors, the project organization is
+a crucial task and the project maintainers are the members that are trusted
+with that task, driving the project, improving the health of the project,
+performing quality control and being responsible over one or more components in
+the project. A Kubernetes maintainer is a contributor who has contributed/is
+contributing to the project regularly and a developer that demonstrated good
+technical judgement. The following is a list of requirements to be a Kubernetes
+maintainer.
+
+Before continuing, it should be noted that we value every contributions, however
+small it may be, even a comment on an issue a valuable contribution to us.
+Maintainership requires time commitment and community trust. You don't have to
+be a maintainer to have a meaningful impact on Kubernetes!
+
+## Maintainership Requirements
+
+In order to become a maintainer, a contributor must satisfy all of the
+requirements listed below.
+
+### 1. Enable 2-Factor Account Authentication for GitHub
+
+See GitHub's documentation about [two-factor authentication](https://help.github.com/articles/about-two-factor-authentication/) for details.
```

Comment:
Could you please send a separate PR to add text about this to governance.md, under the repositories section? All contributors with write or admin access to any Kubernetes repository must enable two-factor auth.